ballerina/build.gradle

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Check other places

Reply:
Fixed in [4e47cc4](https://github.com/ballerina-platform/module-ballerina-yaml/pull/3/commits/4e47cc471807c60f8b218633f36db395cca8d51e)